pkg/controllers/service/service_lb_controller.go

```diff
@@ -55,24 +61,31 @@ func (r *ServiceLbReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	}()
 
 	if err := r.Client.Get(ctx, req.NamespacedName, service); err != nil {
+		if client.IgnoreNotFound(err) == nil {
```

Comment:
maybe we should use
if apierrors.IsNotFound(err) {

Reply:
done